LISISO/install.sh

```diff
@@ -87,6 +81,94 @@ if [[ $distro_version != "5"* ]]; then
  fi
 fi
 
+#Declarations
+
+mlx_driver_size=50
+
+#Clean Installation Temp Disk Usage
+temp_disk_usage_same_partition_clean_install=150
+temp_disk_usage_diff_partition_modules_clean_install=150
+temp_disk_usage_diff_partition_boot_clean_install=20
+
+#Reinstall Installation Temp Disk Usage
+temp_disk_usage_same_partition_reinstall=210
+temp_disk_usage_diff_partition_modules_reinstall=170
+temp_disk_usage_diff_partition_boot_reinstall=18
+
+#Calculations
+rpm_install_size=`rpm -qpi ${distro_package_name}${distro_version}/*hyper-v*x86_64.rpm | grep "Size" | awk '{ total += $3; print }; END { print total /1024/1024}' | tail -1`
```

Comment:
As commented below. If this statement is inside a function and it is called just before install then distro_package_name and distro_version is not required.